### DISCUSSION TITLE
Table Auto Complete

### DISCUSSION THREAD

**Discussion by tdinpsp** [vikramlearning/blazorbootstrap#557] [Ideas]
Sometimes I find it very useful to show more data to the user than just the particular data the user it typing.  For example if the particular data the user is typing is a last name and types "Smith", then there might be many Smith's in the list.  So by showing additional data such as the address and phone number it becomes easy for the user to pick out the exact one that they are looking for.  I've done it before where the control is like a dropdownlist with a key value behind the scenes that is pass back to the program upon the user selected a single item.

**Comment by gvreddy04** [comment]
@tdinpsp Thank you for trying BlazorBootsrtap. I think, this is a good usecase to consider. We will add this in the upcoming releases.

**Comment by tdinpsp** [comment]
One more thing that would be useful is with the Tabs component.  When the user has clicked a tab to be changed add a “Cancel” to the event arguments that would cancel the switch from one tab to another.  The use case is where there are inputs on a tab and the user enters information into them.  Then the user clicks on the new tab without saving the information on the old tab first.  The program needs to check that the user has made some changes and then confirm with the user that they really mean to abandon their changes or if they simply forgot the save them.

I’m able to simulate this now by switching back but it doesn’t look as good to the user because there is a flash of the new tab before it switches back to the old one.


Thanks,

Tarang Deshpande
